repo_validation/tests/validate_srs_consistency/CMakeLists.txt

```diff
@@ -76,6 +76,79 @@ add_dependencies(test_validate_srs_consistency
     test_validate_srs_consistency_clean
     test_validate_srs_consistency_fix
     test_validate_srs_consistency_prefix_preservation
+    test_validate_srs_consistency_malformed_comment
+    test_validate_srs_consistency_line_comment
+    test_validate_srs_consistency_whitespace
 )# Note: We don't add test_validate_srs_consistency_detection as a dependency because 
 # it should fail with exit code 1 (which would fail the build). 
-# Instead, it's available to run manually to verify detection works.
\ No newline at end of file
+# Instead, it's available to run manually to verify detection works.
+# Test 5: Verify malformed comment handling (missing closing bracket)
+add_custom_target(test_validate_srs_consistency_malformed_comment
+    # Create a temporary directory and copy test files
+    COMMAND ${CMAKE_COMMAND} -E make_directory "${CMAKE_CURRENT_BINARY_DIR}/temp_malformed_test"
+    COMMAND ${CMAKE_COMMAND} -E copy_directory
+        "${CMAKE_CURRENT_SOURCE_DIR}/malformed_comment"
+        "${CMAKE_CURRENT_BINARY_DIR}/temp_malformed_test"
+    # Run the script with fix mode
+    COMMAND ${POWERSHELL_EXECUTABLE} -ExecutionPolicy Bypass -File "${VALIDATE_SRS_CONSISTENCY_SCRIPT}"
+        -RepoRoot "${CMAKE_CURRENT_BINARY_DIR}/temp_malformed_test"
+        -Fix
+    # Verify that code was NOT corrupted (all functions still present)
+    COMMAND ${POWERSHELL_EXECUTABLE} -Command "if ((Get-Content '${CMAKE_CURRENT_BINARY_DIR}/temp_malformed_test/test_module.c' -Raw) -notmatch 'TEST_MODULE\* test_module_create\(void\)') { Write-Error 'Function was corrupted/deleted'; exit 1 }"
+    COMMAND ${POWERSHELL_EXECUTABLE} -Command "if ((Get-Content '${CMAKE_CURRENT_BINARY_DIR}/temp_malformed_test/test_module.c' -Raw) -notmatch 'void test_module_destroy\(TEST_MODULE\* module\)') { Write-Error 'Function was corrupted/deleted'; exit 1 }"
+    # Verify that SRS text was actually fixed
+    COMMAND ${POWERSHELL_EXECUTABLE} -Command "if ((Get-Content '${CMAKE_CURRENT_BINARY_DIR}/temp_malformed_test/test_module.c' -Raw) -notmatch 'test_module_create shall allocate memory for a test module') { Write-Error 'SRS text was not fixed'; exit 1 }"
```

Comment:
consider: should we just have an "expected" file that we compare against? This seems oddly targeted and could miss bugs, right?